src/api/app/models/bs_request_action.rb

```diff
@@ -757,9 +757,6 @@ def check_action_permission_source!
 
   def check_action_permission!(skip_source = nil)
     # find objects if specified or report error
-    role=nil
-    sprj=nil
```

Comment:
This can't be removed... it can be undefined (and is trying to use it at line 792 and 823) because at line 776 this can be not assigned if `skip_source` is true